3 - Assignment/240326 - Rust Basic + Roguelike Game, Week 1/collatz-sequence/src/lib.rs

```diff
@@ -1,6 +1,12 @@
 /// Determine the length of the collatz sequence beginning at `n`.
 pub fn collatz_length(mut n: i32) -> u32 {
-    todo!("Implement this")
+    let mut length = 1;
+    while n != 1 {
+        n = if n % 2 == 0 { n / 2 } else { 3 * n + 1 };
+        length += 1;
+    }
+
+    return length;
```

Comment:
`return`을 생략할 수 있습니다.
```suggestion
    length
```